core/src/main/kotlin/br/com/concrete/yosef/api/property/elementgroup/GravityCommand.kt

```diff
@@ -0,0 +1,62 @@
+package br.com.concrete.yosef.api.property.elementgroup
+
+import android.annotation.SuppressLint
+import android.view.Gravity
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import br.com.concrete.yosef.api.property.DynamicPropertyCommand
+import br.com.concrete.yosef.entity.DynamicProperty
+
+/**
+ * Command class that implements the [DynamicPropertyCommand] applying
+ * the orientation property to the view([LinearLayout])
+ *
```

Comment:
Maybe, you should specify here that this command is only possible to be applied if the parent is a FrameLayout